libnopegl/src/utils.h

```diff
@@ -115,6 +115,7 @@ char *ngli_strdup(const char *s);
 int64_t ngli_gettime_relative(void);
 char *ngli_asprintf(const char *fmt, ...) ngli_printf_format(1, 2);
 uint32_t ngli_crc32(const char *s);
+uint32_t ngli_crc32_mem(const uint8_t *s, size_t size);
```

Comment:
Instead of introducing a new function, can we change the prototype of ngli_crc32() instead ?

Reply:
Do we really want to pass through the whole string twice? `ngli_crc32(s, strlen(s))` might not be exactly optimal (I'm thinking of the hash map typically)

Reply:
Ok, makes sense. Should we remame ngli_crc32() to ngli_crc32_str() for consistency ?

Reply:
Not sure I see the need but if you think that's an improvement I don't mind